.travis/deploy_npm

```diff
@@ -36,22 +36,50 @@ fi
 exe node -v
 exe npm  -v
 
+echo ">>> Installing dependencies..."
 exe npm install
-exe npm install --no-save @qooxdoo/compiler
-
-exe npx qx --version
-exe npx qx pkg install -v 
-#compile everything
-exe npx qx compile --config-file=compileServer.json --target=build 
-#exe npx qx compile --config-file=compile.json       --target=build 
-#remove unnecessary stuff
-#exe find ./apps/transpiled -type f -name '*.map' -delete
-#exe find ./apps/transpiled -type f -name '*.js'  -delete
-#copy source
+
+echo ">>> Installing github.com/qooxdoo/qooxdoo-compiler"
+exe [ -d ./qooxdoo-compiler ] && exe rm -rf ./qooxdoo-compiler
```

Comment:
I don't think you want `exe [ -d ./qooxdoo-compiler ] &&`. You're already using `rm -rf` which won't complain if it doesn't exist, and furthermore, since you're test is `-d`, for a directory, if there happened to be a file there called `qooxdoo-compiler` it wouldn't remove it. (Granted, highly unlikely.) In any case, I think you can get rid of the initial test.